.github/workflows/build.yml

```diff
@@ -0,0 +1,73 @@
+name: Build `master`
```

Comment:
@xperiandri no need to rename the workflow file because this still builds & run tests. Also no need to rename workflow name because PRs should run CI too (and PRs are not 'master')

Reply:
The main purpose of having multiple pipelines is the separation of responsibility. You always know which file responsible for what.
Also, I reconsolidated Fornax build logic into the Fake build project. So that the docs build is encapsulated into a build target.
@webwarrior-ws, what do you think?